### DISCUSSION TITLE
RINEX Processing

### DISCUSSION THREAD

**Discussion by gwbres** [nav-solutions/rinex#20] [Ideas]
[This page](https://github.com/gwbres/rinex/blob/main/rinex-cli/doc/processing.md) explains how to operate the CLI for advanced RINEX processing. 

Analysis and algorithms supported to this day

- [x] GNSS recombinations
  - [x] GF recombination
  - [ ] MW combination (best CS detector)
    - [ ] combine NL/WL correctly, we need to form the exact same combinations,
    for the total MW to be correct
- [x] DCBs : Differential code biases estimation
- [ ] MP: code Mutlipath analysis must be concluded

**Comment by larsnaesbye** [comment]
> To do that, we would need two OBS files and one NAV file with shared epochs and content. Is it something you could provide ?

I am quite sure I can do that. Give me a few days :)

**Reply by larsnaesbye** [reply]
There is now a pull request with both NAV and OBS testdata for two stations, same epoch.
